go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/DATA-DOG/go-sqlmock v1.5.0
 	github.com/go-sql-driver/mysql v1.4.0
 	github.com/grafana/grafana-plugin-sdk-go v0.94.0
+	github.com/pkg/errors v0.9.1
```

Comment:
Let's stick to the standard library's `error` package and use `fmt.Errorf("%w: some string", err)` to wrap errors.